src/Umbraco.Web/umbraco.presentation/umbraco/Trees/XmlTree.cs

```diff
@@ -48,7 +47,7 @@ public XmlTree(SerializedTreeType treeType)
 
 		private void Init()
 		{
-            m_JSSerializer = new JSONSerializer { MaxJsonLength = int.MaxValue };
+            m_JSSerializer = new JavaScriptSerializer { MaxJsonLength = int.MaxValue };

```

Comment:
Okay, so I guess I might as well blow it up in the constructor in the example above like:

```csharp
public XmlTree() {
            throw new NotSupportedException("Remove this code, this should never be used");
}
```
Will change and resubmit